### PR TITLE
Update Jobe dockerfile to match current JobeInABox

### PR DESCRIPTION
**Pull request by ascholerChemeketa**
Not needed, but seems worthwhile to update dockerfile while updating Jobe itself.

### CODE REVIEW

File: projects/jobe/Dockerfile [hunk 1]
```diff
@@ -12,59 +12,58 @@ LABEL \
...
+RUN --mount=type=secret,id=api_keys \
+    export API_KEYS=`cat /run/secrets/api_keys | tr '\n' ' '` && \
+    ln -snf /usr/share/zoneinfo/"$TZ" /etc/localtime && \
     echo "$TZ" > /etc/timezone && \
     apt-get update && \
     apt-get --no-install-recommends install -yq \
-    acl \
-    apache2 \
-    build-essential \
-    catch2 \
```
Comment (bnmnetp) [inline projects/jobe/Dockerfile:48]: Do we want to keep this for backward compatibility?  I think we do.

File: projects/jobe/Dockerfile [hunk 2]
```diff
@@ -12,59 +12,58 @@ LABEL \
...
-    python3 \
-    python3-pip \
-    python3-setuptools \
-    sqlite3 \
-    sudo \
-    tzdata \
-    unzip \
-    vim \
-    wget && \
-    python3 -m pip install pylint beautifulsoup4 pandas requests altair && \
```
Comment (bnmnetp) [inline projects/jobe/Dockerfile:67]: definitely need to keep these as some books rely on them.

File: projects/jobe/Dockerfile [hunk 3]
```diff
@@ -12,59 +12,58 @@ LABEL \
     org.opencontainers.image.documentation="https://github.com/trampgeek/jobeinabox" \
     org.opencontainers.image.source="https://github.com/trampgeek/jobeinabox"
 
-ARG TZ=Etc/GMT-0
```
Comment (bnmnetp) [inline projects/jobe/Dockerfile:15]: I think we want to be in GMT so jobe is in the same timezone as the rest of the servers

File: projects/jobe/Dockerfile [hunk 4]
```diff
@@ -12,59 +12,58 @@ LABEL \
...
-    acl \
-    apache2 \
-    build-essential \
-    catch2 \
-    fp-compiler \
-    git \
-    libapache2-mod-php \
-    nodejs \
-    octave \
-    openjdk-18-jdk \
```
Comment (bnmnetp) [inline projects/jobe/Dockerfile:54]: I wonder which version of java we end up with if we use default-jdk???  This could be bad for CSAwesome.

**Review by bnmnetp — CHANGES_REQUESTED** [review]

**Comment by ascholerChemeketa** [comment]
Sorry, that was a sloppy end of the day job.

Started a clean redo and am not 100% sure we want the API keys or other changes. Leaving well-enough alone until if/when we go to jobe 2.x

**Pull request closed without merging**